.gitignore

```diff
@@ -100,4 +100,7 @@ test/apps/test_cuda_hook_static
 test/apps/test_ucp_config
 test/apps/test_memtrack_limit
 test/apps/profiling/ucx_profiling
+test/apps/uct_info/uct_info
+test/apps/uct_info/uct_info_static
+test/apps/uct_info/check_tls.sh
```

Comment:
can remove it after moving check_tls.sh to buildlib/

Reply:
> can remove it after moving check_tls.sh to buildlib/

removed